### DISCUSSION TITLE
My feed is not continuous anymore, limited amount of actitivies

### DISCUSSION THREAD

**Discussion by renanborgez** [community/community#67166] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

With this horrible new feed thing I took a while to customize it in a way that is "similar" to the old feed. Even though it is not perfect now I can not see past events !?!?

Is it a configuration or a bug? I started my day today and I only see 4 activities on my feed even though yesterday my home page was "full".

Is it a limitation or a bug? Because if it is a limitation, that sux by a LOT. 

An easy way to check it is opening the "new" feed and the old feed together

https://github.com/dashboard-feed

You can see that the list of items finishes WAAAAY before than in the old feed.

**Comment by maximebrugel** [comment]
Yes this is horrible...
